index.js

```diff
@@ -114,7 +114,7 @@ function fromObject (that, object) {
 
   if (isArray(object)) return fromArray(that, object)
 
-  if (object == null) {
+  if (object === null || object === undefined) {
```

Comment:
This is [not a performance issue](https://jsperf.com/check-for-null-undefined).
When I made this change, I was under the impression `==` was not allowed in standard.  Apparently it is,  so I'm happy to switch this back.

I personally still prefer the above as a more verbose equivalent, or I prefer the `if (!object)` approach.  Up to you @feross.